### ISSUE TITLE
Provide OSI approval information on license details / summaries

### ISSUE THREAD

**Issue by JonDouglas** [github/choosealicense.com#827]
Would it be helpful to add some type of metadata or OSI stamp to the licenses? The list of licenses from https://spdx.org/licenses/ or https://opensource.org/licenses/alphabetical have columns to provide each license some means of showing it was [OSI Approved](https://opensource.org/licenses). This is a helpful piece of information when looking at a license.

Quick mockups:

![image](https://user-images.githubusercontent.com/1288848/115926666-d7cbf280-a448-11eb-818b-f99d20a53396.png)

![image](https://user-images.githubusercontent.com/1288848/115926749-f5995780-a448-11eb-8a05-3d416d01eeb2.png)

It seems that the [trademark allows for noncommercial/community websites to use it](https://opensource.org/trademark-guidelines).

**Note:** Could additionally do a similar badge or other metadata for the [FSF license list](https://www.gnu.org/licenses/license-list.en.html) which is also on many lists as a column.

**Comment by mlinksva** [comment]
It's an interesting idea. We already have tests looking for OSI (or FSF, or OK) approval; this info could be moved into license metadata (which would itself be tested) and thus rendered. I'd probably suggest text rather than logos.

**Comment by Andre601** [comment]
Maybe a simple SVG logo with hover text could also do the trick? I don't see any real appeal in some text saying stuff like "OSI approved"

**Comment by Aspie96** [comment]
I do not think it is necessary and it would suggest a bias in favour of OSI approval (over, for instance, FSF approval).

If it is added, I think a similar information must also be provided for FSF approval (and possibly GPLv3 compatibility).

**Comment by Tarekchehahde** [comment]
Opened https://github.com/github/choosealicense.com/pull/1366 — approvals.yml generator and OSI/FSF/OD badges.
